cgroups/systemd/apply_systemd.go

```diff
@@ -212,6 +213,17 @@ func (m *Manager) Apply(pid int) error {
 
 	m.Paths = paths
 
+	cpuShares, err := ioutil.ReadFile(path.Join(m.Paths["cpu"], "cpu.shares"))
+	if err != nil {
+		return err
+	}
+
+	if c.CpuShares != 0 {
+		if fmt.Sprintf("%d", c.CpuShares) != strings.TrimSpace(string(cpuShares)) {
+			return fmt.Errorf("The maximum allowed cpu-shares is %s\n", string(cpuShares))
```

Comment:
`\n` at the end is not needed
You don't need `string`, because `fmt` will do this for you.